common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -93,12 +94,11 @@ object PageElement {
         block <- element.textTypeData.toList
         text <- block.html.toList
         element <- Cleaner.split(text)
-        textElement = TextBlockElement(element)
       } yield {
-        if (addAffiliateLinks) {
-          AffiliateLinksCleaner.replaceLinksInElement(textElement, pageUrl = pageUrl, contentType = "article")
-        } else {
-          textElement
+        element match {
+          case _ if element == "<h2>* * *</h2>" => RuleElement(element)
```

Comment:
I don't think this describes everything that `* * *` is supposed to do:

- If immersive
	- It turns it [into a rule](https://github.com/guardian/frontend/blob/master/common/app/views/support/HtmlCleaner.scala#L561) as you have here.
	- but also takes the following paragraph after each *** and wraps the first letter in a [drop cap class](https://github.com/guardian/frontend/blob/master/common/app/views/support/HtmlCleaner.scala#L543) (but only if it matches certain shite)

Unless that's described elsewhere?

It's gross, I don't like it, I'd prefer if there was more structure coming from Composer here or at least was clearer ruling. Feels like it could be a prop on a paragraph component in Dotcom Rendering.

Reply:
So I was silently dropping the immersive requirement here. 

Didn't know about the drop cap, I'll have to think about that more.